Riot/Modules/Room/RoomViewController.m

```diff
@@ -4302,18 +4302,36 @@ - (void)showAdditionalActionsMenuForEvent:(MXEvent*)selectedEvent inCell:(id<MXK
                 
                 [self startActivityIndicator];
                 
-                MXWeakify(self);
-                [self.roomDataSource.room redactEvent:selectedEvent.eventId reason:nil success:^{
-                    MXStrongifyAndReturnIfNil(self);
-                    [self stopActivityIndicator];
-                } failure:^(NSError *error) {
-                    MXStrongifyAndReturnIfNil(self);
-                    [self stopActivityIndicator];
-                    
-                    MXLogDebug(@"[RoomVC] Redact event (%@) failed", selectedEvent.eventId);
-                    //Alert user
-                    [self showError:error];
-                }];
+                // If it's a voice broadcast, delete the selected event and all related events (only if this feature is supported).
+                BOOL supportsRedactionWithRelations = self.mainSession.store.supportedMatrixVersions.supportsRedactionWithRelations || self.mainSession.store.supportedMatrixVersions.supportsRedactionWithRelationsUnstable;
+                if (supportsRedactionWithRelations && selectedEvent.eventType == MXEventTypeCustom && [selectedEvent.type isEqualToString:VoiceBroadcastSettings.voiceBroadcastInfoContentKeyType]) {
```

Comment:
I would not check here the support of the option "withRelations"
I would use the new method by default (the check is done at the sdk level)

Reply:
@giomfo , I pushed this update. We are now using the new endpoint for redaction.